src/core/atom_component/dragTree/component/treeNode/SceneTreeNode.re

```diff
@@ -32,7 +52,12 @@ module Method = {
         checkNodeRelationFunc,
       );
 
-    isTrigger || isAssetWDBFileFunc() ? DragEnter : Nothing;
+    isTrigger || isAssetWDBFileFunc() ?
```

Comment:
TODO refactor(rename)(all: in sceneTree, asset, ...): rename DragEventBaseUtils.re->checkDragEnter,checkDragDrop to isValidForDragEnter/Drop

Reply:
TODO refactor(rename): rename "isTrigger" to "isValid"